src/status_im/multiaccounts/update/core.cljs

```diff
@@ -41,17 +41,35 @@
                        :mnemonic nil
                        {}))
 
+(defn augment-synchronized-recent-stickers
+  "Add 'url' parameter to stickers that are synchronized from other devices.
+   It is not sent from aanother devices but we have it in our db."
+  [synced-stickers stickers-from-db]
+  (mapv (fn [sticker-synced] (assoc sticker-synced :url (as-> stickers-from-db vvv
+                                                          (get vvv (:packID sticker-synced))
+                                                          (:stickers vvv)
+                                                          (filter (fn [sticker-db] (= (:hash sticker-db) (:hash sticker-synced))) vvv)
+                                                          (first vvv)
+                                                          (:url vvv)))) synced-stickers))
```

Comment:
its little bit hard to read, here my suggestion, but it should be tested before merge 
```suggestion
  (mapv #(assoc % :url (-> (get stickers-from-db (:packID %)
                                                          (:stickers)
                                                          (filter (fn [sticker-db] (= (:hash sticker-db) (:hash %))))
                                                          (first)
                                                          (:url))))
                synced-stickers))
```

Reply:
Thanks for the review. I applied your recommendation (except used ->> because filter). It really looks more readable this way. Also tested - worked the same.